bindings/cs/MongoDB.Libmongocrypt/OperationSystemHelper.cs

```diff
@@ -0,0 +1,57 @@
+﻿/*
+ * Copyright 2020–present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#if !NET452
+using System;
+using System.Runtime.InteropServices;
+#endif
+
+namespace MongoDB.Libmongocrypt
+{
+    internal enum OperationSystemPlatform
```

Comment:
nit: I think the enum and class should be named `OperatingSystemPlatform` and `OperatingSystemHelper`.

(i.e "Operating" not "Operation")

Unless I've misunderstood...

Reply:
true, I always thought that `OS` means `Operation System` :( . My fault, done